miasm2/arch/mips32/arch.py

```diff
@@ -60,11 +60,16 @@ def __init__(self, *args, **kargs):
 
 
     @staticmethod
-    def arg2str(e, pos = None):
-        if isinstance(e, ExprId) or isinstance(e, ExprInt):
-            return str(e)
-        assert(isinstance(e, ExprMem))
-        arg = e.arg
+    def arg2str(expr, index=None, symbol_pool=None):
```

Comment:
`index` has been add in all of arg2str call, but seems unused

Reply:
Only renamed